lychee-lib/src/extract/html/html5ever.rs

```diff
@@ -72,6 +72,14 @@ impl TokenSink for LinkExtractor {
                     }
                 }
 
+                // Check and exclude rel=preconnect. Other than prefetch and preload,
+                // preconnect only does DNS lookups and might not be a link to a resource
+                if let Some(rel) = attrs.iter().find(|attr| &attr.name.local == "rel") {
+                    if rel.value.contains("preconnect") {
```

Comment:
Would changing this conditional to the following resolve issue #1499?

`if rel.value.contains("preconnect") || rel.value.contains("prefetch")`